exchange/exchangetest/ccpa-nosale-specific-bidder.json

```diff
@@ -6,17 +6,19 @@
             "site": {
                 "page": "test.somepage.com"
             },
-            "imp": [{
-                "id": "my-imp-id",
-                "video": {
-                    "mimes": ["video/mp4"]
-                },
-                "ext": {
-                    "appnexus": {
-                        "placementId": 1
+            "imp": [
+                {
+                    "id": "my-imp-id",
+                    "video": {
+                        "mimes": ["video/mp4"]
+                    },
+                    "ext": {
+                        "appnexus": {
+                            "placementId": 1
+                        }
                     }
                 }
-            }],
+            ],
```

Comment:
I used a different json formatter, so there will be some of these minor changes. The VS Code Beautify plugin is no longer maintained.